assets/scss/modules/_datatables.scss

```diff
@@ -104,7 +104,11 @@
   .sorting_desc_disabled {
     background: url("../images/sort_desc_disabled.png") no-repeat scroll center right;
   }
-
+  
+  .sorting--hidden {
```

Comment:
only so the sorting arrows don't show when released. 

Alternatively,  I can just apply the sort_xxx_disabled classes